app/controllers/papers_controller.rb

```diff
@@ -91,7 +91,7 @@ def start_review
   def api_start_review
     if params[:secret] == ENV['WHEDON_SECRET']
       @paper = Paper.find_by_meta_review_issue_id(params[:id])
-      if @paper.start_review!(nil, params[:editor], params[:reviewer])
+      if @paper.start_review!(nil, params[:editor], params[:reviewers])
```

Comment:
`reviewers` is now a csv string, i.e. for one review would be `reviewer1` and for two reviewers would be `reviewer1,reviewer2`